components/card/card.css

```diff
@@ -0,0 +1,244 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+.main-container {
+  margin: 2rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  gap: 1rem;
+  align-items: center;
+}
+
+.card {
+  width: 15rem;
+  border-radius:var(--border-md);
+  cursor: pointer;
+  box-shadow: rgba(90, 90, 90, 0.35) 0px 5px 15px;
+  position: relative;
+}
+
+.card-badge {
+  background-color: var(--darkOrange);
+  color: var(--white);
+  position: absolute;
+  display: inline-block;
+  padding: 0.4rem 0.5rem;
+  font-weight: var(--font-semiBold);
+  font-size: 0.8rem;
+  left: 0;
+  top:2%;
+}
+
+.card:hover {
+  box-shadow: rgba(19, 18, 18, 0.35) 5px 15px 15px;
+  transition: 0.3s;
+}
+
+.card-img {
+  border-radius: var(--border-md) var(--border-md) 0 0;
+  max-width: 100%;
+  object-fit: cover;
+}
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
+
+.card-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card-title h3 {
+  font-size: 1rem;
+}
+
+.card-title .close{
+  font-size: 1.5rem;
+}
+.section-main-text .card-description {
+  display: block;
+  color: var(--lightGrey);
+  font-size: 0.7rem;
+  font-weight: 400;
+  padding: 0;
+}
+
+.section-main-text .price p {
+  padding: 0;
+  font-size: 1.1rem;
+}
+
+.price {
+  display: flex;
+  gap: 7px;
+  font-weight: var(--font-semiBold);
+  margin-top: 0.5rem;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+.disc-price {
+  display: inline;
+  padding-right: 0.45rem;
+}
+
+.actual-price {
+  text-decoration: line-through;
+  color: var(--lightGrey);
+  padding-right: 0.45rem;
+}
+
+.price-percentage {
+  color: var(--darkGreen);
+}
+
+.add-cart {
+  width: 95%;
+  margin: 10px 5px;
+}
+
+.add-cart:hover {
+  opacity: 0.9;
+}
+
+/* Horizontal Card Style */
+
+.card-horizontal {
+  width: 100%;
+  display: flex;
+}
+
+.horizontal-img {
+  width: 35%;
+  border-radius: 10px 0 0 0;
+}
+
+.horizontal-btn {
+  width: 100%;
+  border-top: 1px solid rgb(184, 184, 184);
+  display: flex;
+  padding: 1rem 2rem;
+  justify-content: space-evenly;
+}
+
+.horizontal-container {
+  width: 32.5rem;
+}
+
+.qty {
+  margin-top: 1rem;
+  display: flex;
+  flex: flex-wrap;
+  gap: 2px;
+}
+
+.minus,
+.add {
+  color: var(--black);
+  outline: 1px solid var(--lightGrey);
+  border-radius: 50%;
+  padding: 3px 10px;
+  background-color: var(--white);
+}
+
+.minus:hover,
+.add:hover {
+  background-color: rgb(243, 239, 239);
+}
+
+.qty-count {
+  outline: none;
+  width: 30px;
+  padding: 5px 10px;
+  text-align: center;
+  border-radius: 2px;
+  background-color: var(--white);
+  border: 1px solid var(--lightGrey);
+  margin: 0 5px;
+}
+
+
+.horizontal-btn button {
```

Comment:
Avoid grabbing elements in a component library, try using classes in place of it. Doing so can affect the user's styling.